requirements.txt

```diff
@@ -5,3 +5,4 @@ dipy==1.0.0
 svgutils
 nipype==1.2.0
 sentry_sdk==0.13.1
+scikit-learn==0.21.0
```

Comment:
I just added `0.21.0` so that if people who have even older versions of `sklearn` are okay. We can switch to `0.23.0` which seems to be the recent stable release too 👍🏽

Reply:
it was previously set to 0.20.2. https://github.com/PennBBL/qsiprep/blob/master/Dockerfile#L263 Is this too old?

Reply:
No no, I think its completely fine!